Add security policy

pnacht opened the pull request:
Fixes #3709.

This PR adds a simple security policy for OpenH264.

As is, the policy offers two means of reporting a vulnerability: an email (currently a placeholder, I couldn't find an adequate address) or using GitHub's private reporting feature.

Let me know if you'd rather just use one of these.

mcatanzaro commented on SECURITY.md:
I think this project is actually maintained by Cisco employees.

pnacht replied:
Ah, I'd understood from https://www.openh264.org/ that the project had been effectively donated to the open-source community. Will fix.

BenzhengZhang commented:
@pnacht you can add my email address (benzzhan@cisco.com) to the [SECURITY.md]

pnacht commented:
@BenzhengZhang I've added your email to the security policy. I've also gone ahead and removed the option of using GitHub's private reporting tool. If you'd rather I leave it there while you work to get it enabled, let me know and I'll re-add it.
